media-gfx/openscad/openscad-2019.05.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit elisp-common qmake-utils xdg
+
+# 2019.05
+#MY_VER=$(ver_cut 1-2) # version component
+#MY_REL=$(ver_cut 4) # release component, 'p' being the third component
+#MY_PV=${MY_VER}-${MY_REL}
+SITEFILE="50${PN}-gentoo.el"
+
+DESCRIPTION="The Programmers Solid 3D CAD Modeller"
+HOMEPAGE="http://www.openscad.org/"
+SRC_URI="https://github.com/${PN}/${PN}/releases/download/${P}/${P}.src.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="emacs"
+
+# FIXME: add optional lib3mf hidapi spnav
+RDEPEND="
+	dev-cpp/eigen:=
+	dev-libs/boost:=
+	dev-libs/double-conversion:=
+	dev-libs/glib:2=
+	dev-libs/gmp:0=
+	dev-libs/libzip:=
+	dev-libs/mpfr:0=
+	dev-qt/qtconcurrent:=
+	dev-qt/qtcore:=
```

Comment:
Ok, I was thinking it's not necessary because slot 4 has been gone for quite some time.

Do you know of a good way to check whether private headers are used without looking at every single source file and having in-depth knowledge of the Qt files structure?

Reply:
iirc, grep for `include.*{qpa,private,_p}`

Reply:
Looks like no private headers are used.